src/main/java/com/synopsys/integration/detect/workflow/bdio/CodeLocationBdioCreator.java

```diff
@@ -51,11 +69,32 @@ public CodeLocationBdioCreator(final DetectBdioWriter detectBdioWriter, final Si
             final ExternalId externalId = bdioCodeLocation.getDetectCodeLocation().getExternalId();
             final DependencyGraph dependencyGraph = bdioCodeLocation.getDetectCodeLocation().getDependencyGraph();
 
+            // Bdio 1
             final SimpleBdioDocument simpleBdioDocument = simpleBdioFactory.createSimpleBdioDocument(codeLocationName, projectNameVersion.getName(), projectNameVersion.getVersion(), externalId, dependencyGraph);
 
-            final File outputFile = new File(bdioOutput, bdioCodeLocation.getBdioName());
-            detectBdioWriter.writeBdioFile(outputFile, simpleBdioDocument);
-            uploadTargets.add(UploadTarget.createDefault(codeLocationName, outputFile));
+            final File bdio1OutputFile = new File(bdioOutput, bdioCodeLocation.getBdioName() + ".jsonld");
+            detectBdioWriter.writeBdioFile(bdio1OutputFile, simpleBdioDocument);
+            uploadTargets.add(UploadTarget.createDefault(codeLocationName, bdio1OutputFile));
```

Comment:
So we only upload bdio1? Why even make bdio2 a file then? Shouldn't it be the other way around, make bdio1 but upload bdio2?

Reply:
We can't break customers that have older hubs. But we have to create bdio 2 for Polaris

Reply:
We have always taken the stance that Detect only works with the latest Blackduck, so why do this now?